Dockerfile

```diff
@@ -0,0 +1,20 @@
+# Stage1
+FROM node:lts as node
+
+WORKDIR /usr/src/app
+
+COPY package*.json ./
+
+RUN npm ci
+
+RUN npm install -g @angular/cli 
+
+COPY . .
+
+RUN ng build
+
+# Stage 2
+FROM nginx:1.13.12-alpine
```

Comment:
Very old version, any reason why not simply `FROM nginx:stable-alpine` ?

Reply:
Sorry, this was a mistake from copying another docker image I had! 
Will change of course.

Reply:
No worries, this is what community review is for - wondering if I'd missed something where I was using it :-P

Reply:
I have updated the nginx image to the stable image.